app/src/main/java/cm/aptoide/pt/home/HomeFragment.java

```diff
@@ -66,11 +59,9 @@
   private View retryButton;
   private ImageView userAvatar;
   private BottomNavigationActivity bottomNavigationActivity;
+  private LoggedInTermsAndConditionsDialog gdprDialog;
 
-  private PublishSubject<Void> termsAndConditionsAccept;
-  private PublishSubject<Void> termsAndConditionsLogOut;
-  private PublishSubject<Void> termsAndConditionsSubject;
-  private PublishSubject<Void> privacyPolicySubject;
+  private PublishSubject<String> gdprDialogUiSubject;
```

Comment:
This subject could be inside the Dialog class and then a method returning an Observable would expose it to the fragment/presenter.